control-plane/typeSpecConverter.cpp

```diff
@@ -111,6 +115,49 @@ bool TypeSpecConverter::preorder(const IR::Type_Name* type) {
     return false;
 }
 
+bool TypeSpecConverter::preorder(const IR::Type_Newtype* type) {
+    if (p4RtTypeInfo) {
+        bool orig = true;
+        cstring uri;
+        uint32_t sdnB;
+        auto ann = type->getAnnotation(IR::Annotation::pitranslateAnnotation);
+        if (ann != nullptr) {
+            orig = false;
+            int i = 0;
+            for (auto a : ann->body) {
```

Comment:
Did you check this: https://github.com/p4lang/p4c/blob/master/frontends/p4/parseAnnotations.h#L50

Reply:
Sure did, but decided to develop a common check in the new frontend/p4/validatePITranslateAnnotations.h.

Reply:
I checked in code which is now using parseAnnotations.  Please review the final changes.